library/sketches-library/src/main/java/uk/gov/gchq/gaffer/sketches/clearspring/cardinality/serialisation/json/HyperLogLogPlusWithOffers.java

```diff
@@ -26,11 +26,12 @@
 import java.util.List;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Deprecated
 public class HyperLogLogPlusWithOffers {
 
-    private int p = 10;
+    private int p = 5;
 
-    private int sp = 10;
+    private int sp = 5;
```

Comment:
Why is this changing?

Reply:
Default in Gaffer is always HyperLogLogPlus(5, 5) but this class had the wrong defaults set.